opennms-pris-plugins/opennms-pris-plugins-xls/src/main/java/org/opennms/opennms/pris/plugins/xls/source/XlsSource.java

```diff
@@ -31,25 +29,29 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import jxl.Cell;
-import jxl.Sheet;
-import jxl.Workbook;
-import jxl.WorkbookSettings;
-import jxl.read.biff.BiffException;
+
 import org.kohsuke.MetaInfServices;
+import org.opennms.opennms.pris.plugins.xls.source.exceptions.InvalidInterfaceException;
+import org.opennms.opennms.pris.plugins.xls.source.exceptions.MissingRequiredColumnHeaderException;
+import org.opennms.pris.api.InstanceConfiguration;
+import org.opennms.pris.api.Source;
+import org.opennms.pris.model.AssetField;
 import org.opennms.pris.model.PrimaryType;
 import org.opennms.pris.model.Requisition;
 import org.opennms.pris.model.RequisitionAsset;
 import org.opennms.pris.model.RequisitionCategory;
 import org.opennms.pris.model.RequisitionInterface;
 import org.opennms.pris.model.RequisitionMonitoredService;
 import org.opennms.pris.model.RequisitionNode;
-import org.opennms.pris.model.AssetField;
-import org.opennms.pris.api.InstanceConfiguration;
-import org.opennms.pris.api.Source;
```

Comment:
Regarding this arrangements, I've configured IDE as @mvrueden suggested and this was changed after I've configured IDEA to use the OpenNMS code styles and stuff.